From: Mike Gilbert <floppymaster@gmail.com>
Date: Tue, 8 Nov 2016 03:36:41 -0500
Subject: Adjust pkgconfig files to point at rootlibdir (#4584)

The .so symlinks got moved to rootlibdir in 082210c7.
(cherry picked from commit d07903848f7b3b96487e0968f25d88167cc65ded)
---
 Makefile.am                     | 1 +
 src/libsystemd/libsystemd.pc.in | 2 +-
 src/libudev/libudev.pc.in       | 2 +-
 3 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 41c7cfc..ecd8bc1 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -6056,6 +6056,7 @@ EXTRA_DIST += \
 
 # ------------------------------------------------------------------------------
 substitutions = \
+       '|rootlibdir=$(rootlibdir)|' \
        '|rootlibexecdir=$(rootlibexecdir)|' \
        '|rootbindir=$(rootbindir)|' \
        '|bindir=$(bindir)|' \
diff --git a/src/libsystemd/libsystemd.pc.in b/src/libsystemd/libsystemd.pc.in
index e8f7950..7e6d499 100644
--- a/src/libsystemd/libsystemd.pc.in
+++ b/src/libsystemd/libsystemd.pc.in
@@ -7,7 +7,7 @@
 
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=@rootlibdir@
 includedir=@includedir@
 
 Name: systemd
diff --git a/src/libudev/libudev.pc.in b/src/libudev/libudev.pc.in
index a0f3f52..770c922 100644
--- a/src/libudev/libudev.pc.in
+++ b/src/libudev/libudev.pc.in
@@ -7,7 +7,7 @@
 
 prefix=@prefix@
 exec_prefix=@exec_prefix@
-libdir=@libdir@
+libdir=@rootlibdir@
 includedir=@includedir@
 
 Name: libudev
